Source/WebCore/page/scrolling/ScrollingStateScrollingNode.cpp

```diff
@@ -275,6 +276,17 @@ void ScrollingStateScrollingNode::setScrollerImpsFromScrollbars(Scrollbar*, Scro
 }
 #endif
 
+void ScrollingStateScrollingNode::setMouseDidEnterScrollbar(bool flag)
+{
+    if (flag == m_mouseDidEnterScrollbar)
+        return;
+
+#if PLATFORM(MAC)
```

Comment:
/* Unofficial Review */
Do we need this if guard? Other ports do not handle Property::MouseDidEnterScrollbar.

Reply:
Removed.